local-running/compose.yaml

```
@@ -313,3 +313,26 @@ services:
       - "5012:5012"
     volumes:
       - "../../ipv-config/stubs/di-ipv-credential-issuer-stub:/etc/client-configs:ro"
+
+  bav-stub:
+    image: cri-stub:latest
+    container_name: bav-stub
+    build:
+      context: ../../ipv-stubs/di-ipv-credential-issuer-stub
+      dockerfile: Dockerfile
+    environment:
+      - CREDENTIAL_ISSUER_NAME=Bank account verification (Stub)
+      - CREDENTIAL_ISSUER_TYPE=VERIFICATION
+      - CLIENT_AUDIENCE=https://bav-cri.stubs.account.gov.uk
+      - CLIENT_CONFIG_FILE=/etc/client-configs/client-config-core.json
+      - CREDENTIAL_ISSUER_PORT=3012
```

```suggestion
      - CREDENTIAL_ISSUER_PORT=3013
```

Need to have the new CRI stub running on a a new port. 3012 woud clash with the hmrc-kbv stub above